Makefile

```diff
@@ -1,29 +1,30 @@
 .PHONY: test lint run ftest install dev
 
 PYTHON=python3.10
+ARCH=$(shell uname -m)
```

Comment:
Should we have a fallback to set `ARCH` to `framework` if it's not one of the supported values?

Reply:
good idea we can fallback to `x86_64` (also in setup.py)

Reply:
actually just do `make install ARCH=x86_64`